internal/topology/check/compatibility.go

```diff
@@ -35,8 +35,8 @@ func ObjectsAreStrictlyCompatible(current, desired client.Object) field.ErrorLis
 	if current.GetName() != desired.GetName() {
 		allErrs = append(allErrs, field.Forbidden(
 			field.NewPath("metadata", "name"),
-			fmt.Sprintf("cannot be changed from %v to %v to prevent incompatible changes in %v/%v",
-				current.GetName(), desired.GetName(), current.GetObjectKind().GroupVersionKind().GroupKind().String(), current.GetName()),
+			fmt.Sprintf("cannot be changed from %q to %q to prevent incompatible changes in the Cluster",
```

Comment:
Aren't we losing information here? The desired and current names won't give a lot of information to the end user.

Reply:
It feels like we're losing context now with this change. Especially because we don't pass in the parent field path into this func.

Reply:
I have proposed a new version, but the original one was wrong, because incompatible change are in the cluster not in the object being validated